### DISCUSSION TITLE
Github pages not connecting with custom domain

### DISCUSSION THREAD

**Discussion by hamelsmu** [community/community#137749] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've been waiting for this for quite a while, and it doesn't seem to be going through

![image](https://github.com/user-attachments/assets/c11c0a95-f449-4d64-ac9c-acb388b0b63d)

**Comment by LiteBrite82** [comment]
@hamelsmu 
Looks like its up and running now. 
Any remaining issues?

**Comment by maryna-gol** [comment]
If your GitHub Pages site isn’t connecting with your custom domain, there are several steps you can take to troubleshoot and resolve the issue:

### 1. **Check Your DNS Settings**

Ensure that your DNS settings are correctly configured:

- **A Records:** If you're using an A record, make sure it's pointing to GitHub Pages' IP addresses:
  ```
  185.199.108.153
  185.199.109.153
  185.199.110.153
  185.199.111.153
  ```

- **CNAME Record:** If you’re using a CNAME record, it should point to your GitHub Pages domain (`username.github.io`).

**Note:** DNS changes can take some time to propagate (up to 48 hours). 

### 2. **Verify Your GitHub Pages Settings**

- **Repository Settings:** Go to your repository settings on GitHub and check the "Pages" section. Make sure your custom domain is entered correctly and that the DNS settings are correctly set up.

- **Custom Domain Configuration:** Ensure that your custom domain is added correctly in the "Custom domain" field of the GitHub Pages settings.

### 3. **Check SSL/TLS Certificates**

- **HTTPS:** If you’re using HTTPS, make sure your SSL/TLS certificates are correctly configured. GitHub Pages automatically provides HTTPS certificates, but you need to enable the “Enforce HTTPS” option in the GitHub Pages settings if your domain is correctly configured.

### 4. **Verify Domain Configuration**

- **Domain Registrar:** Log into your domain registrar’s dashboard and ensure that the DNS settings are correct. Sometimes, issues can arise if settings are misconfigured at the registrar level.

### 5. **Review GitHub Pages Status**

- **GitHub Status:** Occasionally, GitHub Pages might experience issues. Check GitHub's [status page](https://www.githubstatus.com/) to see if there are any ongoing problems with their service.

### 6. **Clear Browser Cache**

Sometimes, caching issues can cause problems with loading the updated domain settings. Clear your browser cache or try accessing your site from a different browser or device.

### 7. **Check for Domain Configuration Errors**

- **CNAME File:** Ensure that your repository has a `CNAME` file in the root directory with your custom domain name in it. This file is necessary for GitHub Pages to recognize and serve your site at the custom domain.

### 8. **Test DNS Configuration**

- **Online Tools:** Use tools like [DNS Checker](https://dnschecker.org/) to verify that your DNS settings have propagated correctly.
